.circleci/config.yml

```diff
@@ -505,6 +508,16 @@ workflows:
                 - services/shuttle-tide
                 - services/shuttle-tower
                 - services/shuttle-warp
+      - check-standalone:
```

Comment:
Maybe a short comment here about why we need this extra job could be useful?

Reply:
Sure, did that :)